ovos_plugin_manager/language.py

```diff
@@ -52,6 +52,11 @@ def create(config=None):
                 lang_module = OVOSLangDetectionFactory.MAPPINGS[lang_module]
 
             clazz = load_lang_detect_plugin(lang_module)
+            if clazz is None and lang_module != "libretranslate_detection_plug":
```

Comment:
Should this fallback be configurable or at least spec'd in some internal variable?

Reply:
agree, it should read from mycroft.conf (via ovos_utils to account for ovos.conf)

we should also install the default plugin in requirements, probably worth a discussion about what this should be

ideally we want detect and translate to match (so we only add 1 dependency) and to be offline, however there are no good offline translation options

libretranslate is self hosted and makes sense to use, but it introduces latency, maybe detecting could use something like fast lang by default?

Reply:
I like libretranslate as a default since it's free and reliable. I don't think detection should be necessary very often since we should generally have language specified

Reply:
often detection is done before translation to see if it is necessary, arguably whatever is doing that could be refactored to check for `lang`param, 

regardless at OPM level we should assume lang is never present, that is a concern of core/skills

Reply:
True; I think using `lang` when it's available will help performance regardless (both speed and accuracy) but we will need to detect in some casts.. I like reducing default dependencies as much as possible, but I'm fine with adding `fastlang` if there's a performance benefit